shared/constants/types/fs.tsx

```diff
@@ -810,13 +837,12 @@ export type ResetMetadata = {
   resetParticipants: Array<string>
 }
 
-export enum PathItemBadgeType {
-  Upload = 'upload',
+export enum NonUploadPathItemBadgeType {
   Download = 'download',
   New = 'new',
   Rekey = 'rekey',
 }
-export type PathItemBadge = PathItemBadgeType | number
+export type PathItemBadge = UploadIcon | NonUploadPathItemBadgeType | number
```

Comment:
This ends up not being used -- we never badge with upload icon. They only show up as status icon on the left side. But since I already had this, I figured we'd just keep it this way in case it changes in the future.